src/odoc/bin/main.ml

```diff
@@ -935,11 +935,11 @@ end = struct
           exit 1
 
     let generate extra _hidden output_dir syntax extra_suffix input_file
-        warnings_options source_file source_root sidebar =
+        warnings_options source_file source_root sidebar asset_path =
       let source = source_of_args source_root source_file in
       let file = Fs.File.of_string input_file in
       Rendering.generate_odoc ~renderer:R.renderer ~warnings_options ~syntax
-        ~output:output_dir ~extra_suffix ~source ~sidebar extra file
+        ~output:output_dir ~extra_suffix ~source ~sidebar ~asset_path extra file
```

Comment:
I'm not a fan of adding more arguments to this command. This argument makes no sense in most cases and needs to be checked. Similarly, `Document.Asset` seems a bit out of place, as it interacts with nothing else, it could be out of the document thing entirely.

What about adding a different command for generating assets ? `html-generate-asset`, `latex-generate-latex`, etc..

Reply:
The same is true for `--source` and `--source-root`, but OK, I'll do that!

Reply:
(the `Document.Asset` will be inconvenient to remove in my opinion, unless we remove the old way of attaching assets, which would be a good thing I think)